aspnetcore/fundamentals/http-requests.md

```diff
@@ -272,6 +272,95 @@ An `IHttpClientBuilder` is returned when adding named or typed clients. The <xre
 
 [!code-csharp[Main](http-requests/samples/2.x/HttpClientFactorySample/Startup.cs?name=snippet12)]
 
+## Use IHttpClientFactory in a console app
+
+In a console app, add the following package references to the project:
+
+* [Microsoft.Extensions.Hosting](https://www.nuget.org/packages/Microsoft.Extensions.Hosting)
+* [Microsoft.Extensions.Http](https://www.nuget.org/packages/Microsoft.Extensions.Http)
+
+In the following example:
+
+* <xref:System.Net.Http.IHttpClientFactory> is registered in the [Generic Host's](xref:fundamentals/host/generic-host) service container.
+* `MyService` creates a client factory instance from the service, which is used to create an `HttpClient`. `HttpClient` is used to retrieve a webpage.
+* `Main` creates a scope to execute the service's `GetPage` method and write the first 500 characters of the webpage content to the console.
+
+```csharp
```

Comment:
Move this to a file and  bring in as a snippet. How much work would it be to include the entire working sample? 
I think this is all we need now, just this in a snippet that can be easily compiled/updated/etc.

Reply:
Easy ... I'll do it first thing tomorrow morning.